tools/olm-manifests/README.md

```diff
@@ -0,0 +1,11 @@
+# OLM manifests generator
+Bash script for generating OLM manifest files for OperatorHub.
+
+## Requirements
+- bash 5+ (GNU)
+- yq 4.6+ (YAML processor)
+
+## Usage
+```
+./generate-manifests.sh <BUNDLE_VERSION>
```

Comment:
Can you add an example bundle version "eg `./generate-manifests.sh 0.22.1`. Also maybe you should fail the script if someone runs it without the bundle version? Unless producing a bundle called `strimzi-cluster-operator-v` is ok?